src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Properties/InterceptedProjectProperties/ApplicationPropertyPage/ApplicationFrameworkValueProvider.cs

```diff
@@ -29,20 +30,24 @@ internal sealed class ApplicationFrameworkValueProvider : InterceptingPropertyVa
         private const string WinExeOutputType = "WinExe";
         private const string NoneItemType = "None";
         private const string ApplicationDefinitionItemType = "ApplicationDefinition";
+        private const string ApplicationFileName = "Application.xaml";
```

Comment:
From memory, this name varies between C# and VB (because of course it does), so we cannot hard code it.

The `IMyAppFileAccessor` should be used for this I think.

Note that we already pull the full file path as `string? appXamlFilePath  = await GetAppXamlRelativeFilePathAsync(create: true)`, so perhaps you can call `Path.GetFileName` on that string instead?